docs/04.guides/04.cookbooks/24.Query-Handling/page.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Query Handling In Lucee
+id: query-handling-lucee
+---
+
+Here we can find how the lucee supports for SQL queries. 
+
+## Query tags ##
+
+```<cfquery>``` different ways to use the tags in lucee & how we can the passed the value into the query
+
```

Comment:
***toChange***
```<cfquery>``` different ways to use the tags in Lucee and how we can the pass the value into the query